src/components/ShopPage/ShopPageHero/index.tsx

```diff
@@ -0,0 +1,117 @@
+import React from 'react'
+import Rating from "react-rating";
+import Link from 'next/link';
+import Image from "next/image";
+import { useRouter } from "next/router";
+import { toaster } from 'evergreen-ui';
+
+import { EmptyStar, FullStar } from "components/StarRating";
+import { sendEvent } from 'hooks/amplitude';
+import styles from "../../BeachPageHero/styles.module.css";
+
+const ShopPageHero = (props: any) => {
+    const onWebsiteClick = () => {
+        // sendEvent('entry_map__click')
```

Comment:
we should uncomment this and just change them to:

`shop__website__click`
`shop__directions__click`
`shop__contact__click`